automagicallyexecute/task.json

```diff
@@ -32,11 +32,18 @@
       "type": "string",
       "label": "Test Target ID",
       "required": true,
-      "helpMarkDown": "ID for your test target, recevied from us"
+      "helpMarkDown": "ID for your test target, received from us"
+    },
+    {
+      "name": "blocking",
+      "type": "boolean",
+      "label": "Blocking",
+      "required": false,
+      "helpMarkDown": "If your pipeline should wait until all tests are green, default is false"
     }
   ],
   "execution": {
-    "Node16": {
+    "Node20_1": {
```

Comment:
what is the `_1` suffix doing 🤓?

Reply:
Well.... node20 doesn't work I tried that of course. it's also not yet in the schema but "soonTM" see https://github.com/microsoft/azure-pipelines-task-lib/issues/1043

The official how-to then references only 20_1... don't ask me 😬  https://learn.microsoft.com/en-us/azure/devops/extend/develop/add-build-task?view=azure-devops